API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
+description | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+url | UrlField  
+difficulty | CharField (choices)  
+duration | CharField (choices)  
+**Skills** | *:*  
+**WebLitSkills** | *:*  
+**Topic** | 1:*  
+**LessonPlan** | *:*  
```

Comment:
It can be ordered by creation date. But if we want an order, it might be a bit complicated.

@hannahkane Am I correct in saying an activity can belong to multiple modules and a module has multiple activities?

If I am wrong in the above assertion, @alicoding it will actually be 1 to many not many to many.

Either way, @alicoding you can add ordering in a module via a bridge table (or more appropriately, a `through` table that will store the order).